packages/@aws-cdk/aws-msk/lib/cluster.ts

```diff
@@ -830,15 +830,17 @@ export class Cluster extends ClusterBase {
    * Must be using the SASL/SCRAM authentication mechanism.
    *
    * @param usernames - username(s) to register with the cluster
+   * @returns - Array of generated secrets.
    */
-  public addUser(...usernames: string[]): void {
+  public addUser(...usernames: string[]): secretsmanager.Secret[] {
```

Comment:
I'm not sure that this change is the correct approach. I think that I'd recommend making the list of usernames and associated passwords a private property in Cluster and adding a function that takes in a username and returns the associated secret.